eclcli/dns/v2/record.py

```diff
@@ -189,12 +189,30 @@ def get_parser(self, prog_name):
             metavar="<zone_id>",
             help="ID of the zone which recordset you want to list belong to.",
         )
+        parser.add_argument(
+            "--limit",
+            metavar="<limit>",
+            type=int,
+            default=None,
+            help="The number of record sets that you want to display."
+                 "Values range from 1 to 500. (default=100)",
+        )
+        parser.add_argument(
+            "--marker",
+            metavar="<marker>",
+            default=None,
+            help="Specify the ID for the resource."
+                 "It is displayed from the next record of the specified ID.",
+        )
         return parser
 
     def take_action(self, parsed_args):
+        if parsed_args.limit is not None:
```

Comment:
I don't think we should do some validation in take_action. You had better to modify validation function and put it in parser.add_argument as type.